src/c3chart/Area.js

```diff
@@ -13,6 +13,34 @@
         this._type = "area";
     };
     Area.prototype = Object.create(CommonND.prototype);
+    
+    Area.prototype.publish("isStacked", false, "boolean", "Show SubChart");
+    Area.prototype.publish("lineWidth", 1.0, "number", "LineWidth");
+    Area.prototype.publish("dashedLine", [5,5], "array", "Dashed Lines");
+    Area.prototype.publish("lineOpacity", 1.0, "number", "LineWidth");
+
+    Area.prototype.enter = function (domNode, element) {
+        CommonND.prototype.enter.apply(this,arguments);
+    }
+    
+    Area.prototype.update = function (domNode, element) {
+
+        if (this.isStacked()) {
+            this.c3Chart.groups([this._columns.slice(1,this._columns.length)]);
+        } else {
+            this.c3Chart.groups([]);
+        }
+
+        CommonND.prototype.update.apply(this, arguments);
+        this.updateStyles.call(this,element);
```

Comment:
i'm grouping them like this because i need to make sure i call everything after c3 has rendered (otherwise the style attributes get overwritten by c3) this way i know there is a function and that it permeates each class and is the same format across everything. this also allows me to group common.js or commonND publish in there respected files but call it from the [widget].js file